File: mobileApp/app/src/main/java/ru/bmstu/mobileapp/models/Basket.kt
package ru.bmstu.mobileapp.models

data class Basket(
    var id: Int? = null,
    var idInShop: Int? = null,
    var totalAmount: Float? = null,
    var totalAmountWithDiscounts: Float? = null,
    var shopId: Int? = null,
    var items: ArrayList<BasketItem>? = null
)

File: shop/models/Basket.py
#Basket имеет ID и список товаров items
import json


class Basket:
    def __init__(self, idInShop, shopId, callbackURL, items, totalAmount, totalAmountWithDiscounts):
        self.idInShop = idInShop
        self.shopId = shopId
        self.callbackURL = callbackURL
        self.items = items
        self.totalAmount = totalAmount
        self.totalAmountWithDiscounts = totalAmountWithDiscounts

    def toJson(self):
        return json.dumps({
            "idInShop": self.idInShop,
            "shopId": self.shopId,
            "callbackURL": self.callbackURL,
            "items": self.items,
            "totalAmount": self.totalAmount,
            "totalAmountWithDiscounts": self.totalAmountWithDiscounts
        }, default=lambda o: o.__dict__)